src/main/java/sortBench/algorithms/select/SelectSort.java

```diff
@@ -0,0 +1,25 @@
+package sortBench.algorithm
```

Comment:
Here you forgot the `;` at the end of the line

Reply:
working in notpad only is bad....working in eclips is better 

until I won't be a noob.